### PR TITLE
Layer color -- Ready for Review

### PR DESCRIPTION
**Pull request by Evan-CTL**
Currently leaves inactive layers as gray. Pins will turn red and grow slightly when viewing pin details.

### CODE REVIEW

File: media/src/project-activity-components/layers/layer.tsx [hunk 1]
```diff
@@ -36,6 +37,10 @@ export const Layer: React.FC<LayerProps> = (
         setUpdatedLayerTitle(layer.title);
     }, [layer.title]);
 
+    useEffect(() => {
+        setUpdatedLayerColor(layer.color);
+    }, [layer.color]);
+
```
Comment (ndittren) [inline media/src/project-activity-components/layers/layer.tsx:43]: This can probably be moved in with the useEffect in line 36 since they are related

**Comment by ndittren** [comment]
There are more than one migration files here. We can try to consolidate to one by deleting the current ones, running `makemigrations`,  and just pushing that one changed file.

**Comment by Evan-CTL** [comment]
I remade the three migrations into one. Is that what you meant?

**Comment by zmustapha** [comment]
I went through the rendered HTML, and I noticed that the list of colors are not keyboard accessible. The radio buttons default key-navigation is arrow key.
Another thing is that while I can see the color of choice is in the label, e.g. "Color: Amber", the disc isn't highlighted in the list of colors.
I remember styling backgrounds and radio buttons in the past, it's a little tricky. An example is in the Base Map selection for projects. We can go over this on Monday to talk through the keyboard nav and CSS. I was experimenting with the CSS for a bit, I'll continue experimenting a little later.

**Comment by ndittren** [comment]
@Evan-CTL I think the migrations files were moved back in? Should be one file that's next up (should be number 0028) and that one alone should have the changes made in this PR.

**Comment by Evan-CTL** [comment]
I fixed the migration files.

File: media/src/project-activity-components/layers/layer.tsx (hunk 1 above)
Comment (Evan-CTL) [reply media/src/project-activity-components/layers/layer.tsx]: Ugh, re-basing is so frustrating sometimes. Thanks. I took care of it.

**Pull request merged**